common/automation-scripts/OrdersCreation/purchase-order-data-load.sh

```diff
@@ -0,0 +1,172 @@
+#!/bin/bash -e
+
+if [[ $# -lt 2 ]]; then
+   echo "Usage: $0 <db config> <TENANT> [--clear-loan-audit]";
+   echo "       $0 psql.conf diku";
+
+   exit 1;
+fi
+
+export TENANT=${1}
+export orderesNum=${2}
+
+export PGDATABASE=${3}
+export PGUSER=${4}
+export PGPASSWORD=${5}
+export PGHOST=${6}
+export PGPORT=${7}
+
+echo "================================================="
+echo "PGUSER: $PGUSER"
+echo "PGPASSWORD: $PGPASSWORD"
+echo "PGDATABASE: $PGDATABASE"
+echo "PGHOST: $PGHOST"
+echo "PGPORT: $PGPORT"
+echo "TENANT: $TENANT"
+echo "=================================================="
+
+# ==================================================
+#  re-populate the database
+# ==================================================
+
+
+
+psql -a -c "SELECT count(*) FROM ${TENANT}_mod_orders_storage.purchase_order "
+
+while IFS=, read -r organizationID AccNum
+do
+psql -a -c "CREATE OR REPLACE FUNCTION public.generate_data_for_edifact_export(organizations_amount integer,
+                                                                   orders_per_vendor integer,
+                                                                   polines_per_order integer) RETURNS VOID as
+\$\$
+
+DECLARE
+    -- !!! SET DEFAULT TENANT NAME !!!
+    orgName   text DEFAULT 'perf_test_vendor';
```

Comment:
Can these be any strings?  If so, can you put a comment maybe in line 44 about this?